src/components/Card/CardItem.tsx

```diff
@@ -14,35 +14,34 @@ export interface CardItemData {
 
 export interface CardItemProps {
   data: CardItemData;
-  // ref: RefObject<string>;
   handleSwipe: (direction: string) => void;
   handleLeftScreen: (direction: string) => void;
 }
 
 export function CardItem({
   data,
-  // ref,
   handleSwipe,
   handleLeftScreen,
 }: CardItemProps) {
   const { link, imgSrc, title, text } = data;
 
   return (
     <TinderCard
-      // ref={ref}
       className="swipe"
       onSwipe={handleSwipe}
       onCardLeftScreen={handleLeftScreen}
       swipeRequirementType="position"
     >
-      <Card role="listitem" className="card-item user-select-none">
-        <Card.Img variant="top" src={imgSrc} draggable={false} />
-        <Card.Body>
+      <Card className="user-select-none card-item" role="listitem">
+        <Card.Img className="card-img-top" src={imgSrc} draggable={false} />
+        <Card.Body className="d-flex flex-column">
           <Card.Title>{title}</Card.Title>
           <Card.Text>{text}</Card.Text>
-          <Card.Link href={link} target="_blank">
-            {link}
-          </Card.Link>
+          <Card.Footer className="mt-auto">
+            <Card.Link href={link} target="_blank">
```

Comment:
In the mobile view (both on my actual phone and when using Chrome's dev tools), when I click this link, the color changes slightly but no new tab opens up.

On desktop, clicking the link opens the URL in a new tab, as expected.